.devcontainer/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM mcr.microsoft.com/devcontainers/python:3.11-bullseye@sha256:105bf6a63ab025206f019a371a735fec6553db0be520030c7a2fd0e002947232
```

Comment:
These appear to be hosted by Microsoft instead of on Docker Hub. I'm having trouble finding this exact SHA, though I found the list of Dev Container Python image tags: https://mcr.microsoft.com/v2/devcontainers/python/tags/list. Where can I find this?

Reply:
https://mcr.microsoft.com/en-us/product/devcontainers/python/tags and search for the sha

Reply:
To look this up, I of course had to do the reverse search. I went to https://mcr.microsoft.com/en-us/product/devcontainers/python/tags and searched for the `3.11-bullseye` tag.